tools/unix/create-internal-pr.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+# This tool will open a PR to update workerd in Cloudflare's internal repository
+# to match the current branch.
+
+set -euo pipefail
+
+BRANCH_NAME=$(git rev-parse --abbrev-ref HEAD)
+SYNC_BRANCH_NAME=sync/$BRANCH_NAME
+
+cd ../edgeworker
+git fetch origin master
+
+if git rev-parse --verify $SYNC_BRANCH_NAME; then
+	BRANCH_EXISTS=1
+else 
+	BRANCH_EXISTS=0
+fi
+
+
+if [ $BRANCH_EXISTS -eq 1 ]; then
+	git checkout $SYNC_BRANCH_NAME
+else
+	git checkout master 
+	git checkout -b $SYNC_BRANCH_NAME
+fi
+
+git -C deps/workerd fetch origin $BRANCH_NAME
```

Comment:
You can rely on `gh` cli tool here. I mostly do `gh pr checkout PR_ID` whenever I navigate to deps/workerd.

Reply:
Yeah I use it too but I wanted the script to work without it.